.pep8speaks.yml

```diff
@@ -0,0 +1,23 @@
+# File : .pep8speaks.yml
+
+message:  # Customize the comment made by the bot
+    opened:  # Messages when a new PR is submitted
+        header: "Hello @{name}, Thank you for submitting the Pull Request !"
+                # The keyword {name} is converted into the author's username
+        footer: "If you have not done so, please consult the [`jwql` Style Guide](https://github.com/spacetelescope/jwql/blob/master/style_guide/style_guide.md)"
+                # The messages can be written as they would over GitHub
+    updated:  # Messages when new commits are added to the PR
+        header: "Hello @{name}, Thank you for updating !"
+        footer: ""  # Why to comment the link to the style guide everytime? :)
+    no_errors: "Cheers ! There are no PEP8 issues in this Pull Request. :beers: "
+
+scanner:
+    diff_only: False  # If True, errors caused by only the patch are shown
+
+pycodestyle:
+    max-line-length: 200  # Default is 79 in PEP8
+    ignore:  # Errors and warnings to ignore
+        - W391
```

Comment:
Can we put a comment here about what this warning is? (What is this warning? lol)

Reply:
@SaOgaz This warning refers to not having a blank line at the end of the file, which on second thought, I decided that I didn't want to ignore afterall, so I removed it.